src/core/localization/gettext/ar/common.po

```diff
@@ -8,491 +8,491 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
 
-#: km
+#: km
 msgid "km"
 msgstr "كم"
 
-#: m
+#: m
 msgid "m"
 msgstr "م"
 
-#: to
+#: to
 msgid "to"
 msgstr "إلى"
 
-#: logout
+#: logout
 msgid "Logout"
 msgstr "تسجيل خروج"
 
-#: save
+#: save
 msgid "Save"
 msgstr "حفظ"
 
-#: cancel
+#: cancel
 msgid "Cancel"
 msgstr "إلغاء"
 
-#: create
+#: create
 msgid "Create"
 msgstr "إنشاء"
 
-#: disasters
+#: disasters
 msgid "Disasters"
 msgstr "كوارث"
 
-#: loading
+#: loading
 msgid "Loading..."
 msgstr "جارٍ التحميل..."
 
-#: loading_events
+#: loading_events
 msgid "Loading events"
 msgstr "جارٍ تحميل الأحداث"
 
-#: legend
+#: legend
 msgid "Legend"
 msgstr "عنوان تفسيري"
 
-#: maps
+#: maps
 msgid "maps"
 msgstr "خرائط"
 
-#: vertical_direction
+#: vertical_direction
 msgid "Vertical direction"
 msgstr "الاتجاه الرأسي"
 
-#: horizontal_direction
+#: horizontal_direction
 msgid "Horizontal direction"
 msgstr "الاتجاه الأفقي"
 
-#: legend_presentation
+#: legend_presentation
 msgid "Legend presentation"
 msgstr "عرض العنوان التفسيري"
 
-#: layers bivariate##color_manager##layers_filter
+#: layers
 bivariate##color_manager##layers_filter

```

Comment:
hm, it's not ok. 
how it happened? after i18n:gettext-sync ? 
so after I run this command on your branch - result is next: 
<img width="1792" alt="Screenshot 2023-02-10 at 18 11 17" src="https://user-images.githubusercontent.com/20676525/218126443-38edb3a8-4730-435d-9d28-91694948e122.png">

Reply:
what is your msgmerge version? 
msgmerge --version
msgmerge (GNU gettext-tools) 0.21

Reply:
same version as yours
![image](https://user-images.githubusercontent.com/50058726/218148170-25654274-d078-459b-870b-c4c06b4bd0d4.png)

I think it was from sync, lesser possibility - from update or export commands for i18n